doc/nrf/device_guides/working_with_fem.rst

```diff
@@ -15,80 +15,73 @@ Some FEMs, like the nRF21540, also provide a power down (PDN) control that power
 
 For testing purposes, a FEM is usually integrated in either a development kit or a shield that you can connect to a development kit.
 
-This guide describes how to add support for 2 different front-end module (FEM) implementations to your application in |NCS|.
+This guide describes how to add support for different front-end module (FEM) implementations to your application in |NCS|.
 
 .. _ug_radio_fem_sw_support:
 
 Software support
 ****************
 
-To use radio protocols and a FEM with your application, enable :ref:`nrfxlib:mpsl_fem` in the :ref:`nrfxlib:mpsl` (MPSL) library.
-
-The following test samples support FEM control:
-
-* :ref:`radio_test`
-* :ref:`direct_test_mode`
-
-You can also use your own FEM driver when required.
-
-.. _ug_radio_fem_sw_support_mpsl:
-
-Using MPSL
-==========
-
-The MPSL library provides the following GPIO interface implementations:
+The following GPIO interface implementations are available in |NCS|:
 
 * :ref:`ug_radio_fem_nrf21540_spi_gpio` - For the nRF21540 GPIO+SPI implementation that uses a 3-pin interface and an SPI interface with the nRF21540.
 * :ref:`ug_radio_fem_nrf21540_gpio` - For the nRF21540 GPIO implementation that uses a 3-pin interface with the nRF21540.
 * :ref:`ug_radio_fem_skyworks` - For the implementation that uses a 2-pin interface with the SKY66112-11 device and other compatible front-end modules.
 
-To use these implementations, your application must use a protocol driver that enables the FEM feature.
+To use any of these implementations with your application, the following conditions must be satisfied:
 
-The library provides multiprotocol support, but you can also use it in applications that require only one protocol.
-To avoid conflicts, check the protocol documentation to see if the protocol uses the FEM support provided by MPSL.
+1. The :ref:`nrfxlib:mpsl_fem` in the :ref:`nrfxlib:mpsl` (MPSL) library must be enabled. It provides implementations of drivers for supported Front-end modules.
 
-.. note::
-   Currently, the following protocols use the FEM support provided by MPSL:
+   * Enable support for MPSL by setting the :kconfig:option:`CONFIG_MPSL` Kconfig option to ``y``. For radio protocols drivers based on MPSL, this option is selected by default.
+   * Enable support for the FEM subsystem by setting the :kconfig:option:`CONFIG_MPSL_FEM` Kconfig option to ``y``. This option is selected automatically if a supported FEM is provided in the application's devicetree file.
+
+   .. note::
+      If the application cannot use MPSL but wishes to use only the FEM driver provided by MPSL, it can be achieved by enabling the following Kconfig options:
+
+      * :kconfig:option:`CONFIG_MPSL`
+      * :kconfig:option:`CONFIG_MPSL_FEM_ONLY`
+
```

Comment:
There is whole chapter about this "MPSL FEM-only configuration", no need to duplicate. If needed please link to this chapter.